src/services/user.ts

```diff
@@ -256,12 +255,13 @@ const setHomePhonePrivacy = (makePrivate: boolean) =>
 const setImagePrivacy = (makePrivate: boolean) =>
   http.put('profiles/image_privacy/' + (makePrivate ? 'N' : 'Y')); // 'Y' = show image, 'N' = don't show image
 
-const getBirthdate = async (): Promise<DateTime> =>
-  DateTime.fromISO(await http.get('profiles/birthdate'));
+const getBirthdate = async (): Promise<Date> => await http.get('profiles/birthdate');
 
 const isBirthdayToday = async () => {
-  const birthday = await getBirthdate();
-  return birthday?.month === DateTime.now().month && birthday?.day === DateTime.now().day;
+  const birthday = new Date(await getBirthdate());
+  return (
+    birthday.getMonth() === new Date().getMonth() && birthday.getDate() === new Date().getDate()
+  );
 };
```

Comment:
Since we have committed to using `date-fns` for out date utilities in 360, it would make sense to use their `isToday function to simplify here:
```suggestion
const isBirthdayToday = async () => {
  const birthdate = new Date(await getBirthdate());
  return isToday(birthdate);
};
```

In fact, you could then fit this onto one line, and more importantly make (in my opinion) more expressive and clear:
```suggestion
const isBirthdayToday = (): Promise<boolean> => getBirthdate().then(isToday)
```